src/components/Header/Header.js

```diff
@@ -16,7 +16,7 @@ class Header extends Component {
                     </button>
                     <div className="navbar-brand">
                       <img src={KrantiLogo} className="header_logo" alt="logo"/>
-                      <a className="logo_title" href="#home">Kranti</a>
+                      <a className="logo_title" href="/public/index.html">Kranti</a>
```

Comment:
Here the title 'Kranti' is directing to the home page and not the logo.

Reply:
@devshiva619, This is not the way to change the pages in React.js. 
React.js uses React-Router to change the links and pages. 
For Homepage there will be a route defined as `<Route path="/" exact component={Homepage} />`. 
You have to follow the React.js way to define the routes.